### DISCUSSION TITLE
demo code not working

### DISCUSSION THREAD

**Discussion by szwang-seedtech** [dipy/dipy#2935] [3. Community Lounge]
the demo code in 1.4.0 gives no results.

The current website seems not working properly, so I used the 1.4.0 website, and run some sample code from there. However, the operation of some scripts, such as basic fiber tracking, gave me empty results.

I was using the dipy docker image version 1.4.1 and fury version 0.7.0.

**Comment by skoudoro** [comment]
Hi @szwang-seedtech,

There is no official docker from DIPY, so not sure which docker you are talking about.

We will need more details to understand your issue. Can you explain each steps ? how did you run the basic tracking with docker? not sure to understand all your steps.

Concerning the website, it will be updated by the end of this month, we are currently working on it. 

Thank you.

**Reply by szwang-seedtech** [reply]
Hello, @skoudoro thank you for reply!

I was just follow the steps on this example: https://dipy.org/documentation/1.6.0./examples_built/tracking_introduction_eudx/#example-tracking-introduction-eudx

The docker image I used is "brainlife/dipy", and I believe it is the "window.scene" that blocks the operation of the scripts. Before or without the function "window.scene", everything works fine. But once with it, the operation of the scripts will stop right on it.

**Reply by szwang-seedtech** [reply]
I figured out the reason. Thank you for your concern all the same.

**Reply by staunton-golding** [reply]
hi, what was your solution, I am having the same issues!
